example/spec/ingestGranule/IngestGranuleDuplicateHandlingVersionSpec.js

```diff
@@ -0,0 +1,212 @@
+'use strict';
```

Comment:
I think there are some things here that we can do differently and not put so much on the integration test system. I think for duplication granule handling, we should do the full integration tests on the `version` and `catchable error` scenarios. The other cases should be done in the unit tests or an alternative AWS solution would be to have a test that has the data in place and just invokes the one lambda.

Running 3 ingest workflows for each (4) scenarios is adding a lot of time to the integration test suite, but not giving us that much for the amount of time, so I think we should consider how to test this with less runs.